pkg/logs/input/kubernetes/launcher.go

```diff
@@ -206,9 +206,6 @@ func (l *Launcher) getSource(pod *kubelet.Pod, container kubelet.ContainerStatus
 	if err := cfg.Validate(); err != nil {
 		return nil, fmt.Errorf("invalid kubernetes annotation: %v", err)
 	}
-	if err := cfg.Compile(); err != nil {
```

Comment:
Why are we removing this?

Reply:
move it to validate, https://github.com/DataDog/datadog-agent/pull/3022/files#diff-d35fdb7f310c415e02d1bb852fb270e9R68